components/utility/Scanner.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState, useEffect, useRef } from "react";
+import { BrowserMultiFormatOneDReader, IScannerControls } from "@zxing/browser";
+
+type Device = {
```

Comment:
can you put this type somewhere in lib/types?

Reply:
Yup I see you did it, will move it there